app/src/main/java/org/thunderdog/challegram/data/TGChat.java

```diff
@@ -762,8 +783,11 @@ public int getCurrentWidth () {
   }
 
   private void setCounter (boolean allowAnimation) {
+    boolean hasReactions = chat != null && chat.unreadReactionCount > 0;
```

Comment:
Unread reaction badge should be displayed for archive as well (similar how to mention badge is displayed).